portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,216 @@
 <!DOCTYPE html>
-<html>
-  <head>
+<html lang="en">
+
+<head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SAM | PORTFOLIO</title>
+    <link rel="icon" href="images/favicon.ico" size="16x16" type="image/x-icon" />
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+    <div id="navbar">
+        <h1><a href="#home" class="scroll">Samantha Murray Tuesta</a></h1>
+        <nav>
+            <ul>
+                <li><a href="#home" class="scroll">Home</a></li>
+                <li><a href="#about" class="scroll">About</a></li>
+                <li><a href="#projects" class="scroll">Projects</a></li>
+                <li><a href="#social" class="scroll">Social</a></li>
+            </ul>
+        </nav>
     </div>
-  </body>
+
+    <section id="home">
+        <svg class="home-blob" viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg">
```

Comment:
In general, I'd recommend having SVGs as external files because they tend not to be edited by hand.

Reply:
Gotcha! In this case, since I am manipulating the css to have a fill of an image or even of a variable from the root as the set color for fill or stroke I think the use of html allows for easier css application! Do you have a recommended approach for the application of css or would it be alright to continue using the method that I have implemented?